content/using/install.md

```diff
@@ -457,11 +279,10 @@ Galaxies are rare and extremely valuable, meaning that there is no liquid market
 
 As mentioned before, stars are valuable. You are solely responsible for safeguarding your seeds and secret keys, and arming yourself with the computer-security knowledge that is attendant upon that safeguarding. You act at your own risk in reliance upon the contents of the manual. In no way is the Tlon Corporation responsible for the actions, decisions, or other behavior taken or not taken by you in reliance upon this manual.
 
-It’s a good idea to store your keys redundantly; for example, on both a USB stick and a piece of paper in the safe, in case one of those methods fails. If you deem a key to be valuable enough, you can shard it into thirds and store each third in secure, geographically distributed locations. Consider these storage media options:
+It’s a good idea to store your keys redundantly in case one method fails. If you deem a key to be valuable enough, you can shard it into thirds and store each third in secure, geographically distributed locations. Consider these storage media options:
 
-- Printing the secret on a piece of paper. However, paper wallets are vulnerable to various forms of physical damage, such as rot, water damage, smoke, or fire. Laminating the paper can mitigates some of these risks, but the lamination can potentially trap moisture. Make sure that you trust the printer: some have memory and network connections.
-- Storing the secret on a brand-new USB stick or hardware wallet (such as a Ledger) that is never connected to a networked machine.
-- Engraving the secret on a strip of stainless steel. This medium is resistant to both water damage and fire damage.
+- Storing the secret on a hardware wallet (such as a Trezor or Ledger) that is never connected to a networked machine.
```

Comment:
Why was paper removed?

Reply:
I've read paper is just generally problematic (often destroyed/lost), I can add it back if you think we should have it.

Reply:
I opted not to add paper back here (I thought of adding it with a flood/fire caveat - I've just read a lot of disaster stories of keys lost on paper), seems like if we're going to suggest things we might as well suggest the two better options.